fvm/env.go

```diff
@@ -109,10 +108,6 @@ func (env *commonEnv) LimitAccountStorage() bool {
 	return env.ctx.LimitAccountStorage
 }
 
-func (env *commonEnv) VM() *VirtualMachine {
-	return env.vm
-}
-
 func (env *commonEnv) BorrowCadenceRuntime() *ReusableCadenceRuntime {
 	return env.ctx.ReusableCadenceRuntimePool.Borrow(env.fullEnv)
```

Comment:
maybe we can emit some metrics in `Borrow` and `Return` func for monitoring purposes on Grafana? sth as simple as current active borrowed Cadence runtime etc. WDYT?

Reply:
I'm planning to refactor the borrowing to only happen once per transaction/script.  Instead of passing in the vm to Run (https://github.com/onflow/flow-go/blob/master/fvm/fvm.go#L18), we will pass in the ReusableCadenceRuntime.  We won't need the extra metric in that case.